internal/cache/util/util_test.go

```diff
@@ -85,17 +86,18 @@ func (ut *utilTest) Test_CreateFile_FileDirNotPresent() {
 
```

Comment:
I think we should add test case where file dir is not present and we try to create file with dirPerm < 600

Reply:
Done